skiko/src/iosMain/kotlin/org/jetbrains/skiko/SkiaLayer.ios.kt

```diff
@@ -24,6 +24,16 @@ actual open class SkiaLayer {
         return true
     }
 
+    fun showScreenKeyboard() {
```

Comment:
How it was tested? If just do 
```
window!!.rootViewController = SkikoViewController(
            SkikoUIView(
                SkiaLayer(SkikoGestureEventKind.values()).apply {
                    skikoView = GenericSkikoView(this, makeApp(this))
                    showScreenKeyboard()
                }
            )
        )
```
in sample keyboard is not  shown.

Reply:
We should show/hide keyboard only after view is loaded, so code should be smth like this:
```
override fun application(application: UIApplication, didFinishLaunchingWithOptions: Map<Any?, *>?): Boolean {
        window = UIWindow(frame = UIScreen.mainScreen.bounds)
        var showKeyboard: () -> Unit = {}
        window!!.rootViewController = SkikoViewController(
            SkikoUIView(
                SkiaLayer(SkikoGestureEventKind.values()).apply {
                    skikoView = GenericSkikoView(this, makeApp(this))
                    showKeyboard = { showScreenKeyboard() }
                }
            )
        )
        window!!.makeKeyAndVisible()
        showKeyboard.invoke()
        return true
    }
```

PS. Also make sure the option I/O -> Keyboard -> Connect Hardware Keyboard is unset.

Reply:
Yep, works this way, modulo backspace key.